app.js

```diff
@@ -1,14 +1,26 @@
 const express = require("express");
+const fs = require("fs").promises;
 const app = express();
-const { getAllTopics } = require('./controllers/topics');
+const { getAllTopics } = require("./controllers/topics");
+const endpoints = require("./endpoints.json");
 
 app.use(express.json());
 
-app.get("/api/topics",getAllTopics );
+app.get("/api", (req, res) => {
+ 
+  res.status(200).send(endpoints);
+});
```

Comment:
The implementation here is fine, but think it would be better if that logic was extracted to a controller.